static/skins/historic/main.css

```diff
@@ -0,0 +1,308 @@
+@import url('http://fonts.googleapis.com/css?family=Courgette&subset=latin,latin-ext');
```

Comment:
All of those are OFL, so we can redistribute. Please download the fonts, put them into static files and refer from there. 

`googleapis.com` is often blocked, it's another DNS lookup and we'll set up our own CDN caching.